.gitlab-ci.yml

```diff
@@ -120,6 +120,17 @@ test:
   script:
     - cargo test --verbose --all-features --no-fail-fast --workspace
 
+
+docs:
+    stage:                           workspace
+    <<:                              *docker-env
+    script:
+        - output=$( cargo doc --workspace --no-deps 2>&1 )
+        - 'has_warnings=$(egrep -c "^warning: [1-9]+[0-9]* warnings emitted" <<< "$output" || true)'
```

Comment:
I need to wrap the line in `'…'` because the colon is otherwise interpreted by GitLab as the start of some mapping.

Reply:
yeah, this is so dumb